dvr_scan/video_joiner.py

```diff
@@ -84,8 +85,20 @@ def position(self) -> FrameTimecode:
         """Current position of the video including presentation time of the current frame."""
         return self._position + 1
 
+    @property
+    def position_ms(self) -> float:
+        return self._cap.position_ms
+
+    @property
+    def last_position_ms(self) -> float:
+        return self._last_position_ms
+
     def read(self, decode: bool = True) -> Optional[numpy.ndarray]:
         """Read/decode the next frame."""
+
+        #  I get the feeling when you run off the end of a file the position_ms returns 0.
```

Comment:
Could we add a unit test to verify this?  You can [add one to `tests/test_video_joiner.py`](https://github.com/Breakthrough/DVR-Scan/blob/develop/tests/test_video_joiner.py).